src/main/java/com/github/zly2006/reden/RedenClient.kt

```diff
@@ -159,6 +159,19 @@ class RedenClient : ClientModInitializer {
                         relaunch(null)
                         1
                     }
+                    literal("neofetch") {
+                        literal("chars").executes {
+                            val renderer = MinecraftClient.getInstance().textRenderer
+                            val width = renderer.getWidth(" ")
+                            (0x20..0x7E)
+                                .map { it.toChar() }
+                                .groupBy { renderer.getWidth(it.toString()) }
+                                .forEach {
+                                    println("${it.key}: ${it.value.joinToString("")}")
+                                }
+                            1
+                        }
+                    }
```

Comment:
What is this command used for?

Reply:
get chars that have the same width to generate ascii art